src/Components/CreateImageWizardV2/steps/Review/Footer/Footer.tsx

```diff
@@ -33,6 +35,10 @@ const ReviewWizardFooter = () => {
       reset: resetCreate,
     },
   ] = useCreateBlueprintMutation({ fixedCacheKey: 'createBlueprintKey' });
+
+  // initialize the server store with the data from RTK query
+  const oscapData = useOscapData(useAppSelector(selectProfile));
+  const serverStore = { ...oscapData };
```

Comment:
This is my suggestion for reducing data duplication, instead of storing and mange the same data twice in redux store and RTK query, we can reuse the data from RTK and create a `ServerStore`